hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/OperationBuilder.java

```diff
@@ -765,4 +790,9 @@ public T valueAttribute(String value) {
         return new WriteAttributeOperationBuilder<>(null);
     }
 
+    @SuppressWarnings("unchecked")
+    public static GenericOperationBuilder<GenericOperationBuilder<?>> generic(String operationName) {
```

Comment:
@jmazzitelli that `generic()` name was my spontaneous suggestion and I do not find it ideal myself. Let's find a better name.

As @josejulio points out, the sibling methods in this class are called 1:1 after the operation whose builder they return. This one returns a generic operation builder by name. 

> Couldn't this method be called "operation" or "executeOperation" or "operationName" to be explicit what its for?

From your proposals, I find `operationName` the best. 

My proposals are `OperationBuilder.byName("reload")...` or `OperationBuilder.byOperationName("reload")...` 
WDYT?